manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -137,6 +137,14 @@ sidecarInjectorWebhook:
   #
   # defaultTemplates: ["sidecar", "hello"]
   defaultTemplates: []
+
+  # defaultExcludedNamespaces specifies a set of namespaces that sidecar injection will be skipped.
```

Comment:
Should this also extend to not pushing out the `istio-ca-root-cert` configmap to these namespaces as well?

Reply:
Good point, I'll update this if the PR is reasonable to be merged. Also, I realized there may be more places that need to be revised.